packages/app/src/cli/utilities/extensions/fetch-product-variant.ts

```diff
@@ -12,12 +14,18 @@ export async function fetchProductVariant(store: string) {
   const adminSession = await ensureAuthenticatedAdmin(store)
   const result: FindProductVariantSchema = await adminRequest(FindProductVariantQuery, adminSession)
   const products = result.products.edges
-  if (products.length === 0)
+  if (products.length === 0) {
+    const normalizedUrl = `https://${await normalizeStoreFqdn(store)}/admin/products/new`
```

Comment:
I've heard that using `/admin` can create quirks with Identity and should not be used, instead we should use `admin.shopify.com` for all new code. Was it @alxclark who mentioned it maybe?

Reply:
We don't have a function to convert foo.myshopify.com to admin.shopify.com/store/foo .  Am sure it's easy enough to add. There are likely a lot of places in CLI and in other repos, as well as a lot of 3P code, that aren't doing this though.

Reply:
Yeah I think @shaun is right that this is something that should be addressed separately as I'm seeing various `/admin` URLs throughout the CLI.